modules/apps/frontend-taglib/frontend-taglib-clay/src/main/resources/META-INF/resources/management_toolbar/SearchControls.js

```diff
@@ -40,13 +40,14 @@ const SearchControls = ({
 				<ClayInput.Group>
 					<ClayInput.GroupItem>
 						<ClayInput
+							aria-label={`${Liferay.Language.get('search')}:`}
```

Comment:
Does the colon have any significance? :thinking: Is there a difference for a screen reader between this and just 
```html
aria-label={Liferay.Language.get('search')}
```

Reply:
Hi @markocikos! Yes, screen readers detect punctuation symbols and each user can have a specific setting for that.